Tella/Scenes/Settings/ViewModel/UwaziServerViewModel.swift

```diff
@@ -242,6 +243,9 @@ class UwaziServerViewModel: ObservableObject {
                     self.shouldShowLoginError = true
                     self.loginErrorMessage = error.localizedDescription
```

Comment:
I think it's better to display `error.errorDescription` and I suggest to be the "errorDescription" not optional and have a default value like `unexpectedResponse`

Reply:
As we are implementing the protocol `LocalizedError` the `errorDescription` is variable defined  which is an optional we cannot make it non optional.